components/BottomNavigation/BUILD

```diff
@@ -24,13 +24,37 @@ load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 
 licenses(["notice"])  # Apache 2.0
 
-mdc_public_objc_library(
+mdc_objc_library(
```

Comment:
😢

Reply:
Is this change because we can't use `mdc_public_objc_library` here or because we're intentionally moving away from it?